orca/requester.py

```diff
@@ -1,48 +1,159 @@
 """Module responsible for handling requests to data servers"""
 
 import logging
+import requests
 from xarray import open_mfdataset, open_dataset
 import re
-from math import ceil
 import os
 from datetime import datetime
 
 
 logger = logging.getLogger("scripts")
 
 
-def file_from_opendap(url, outdir, outfile):
+def file_from_opendap(url, threshold, outdir, outfile):
     """Write to file from OPeNDAP link"""
-    urls = bisect_request(url)
+    input_dataset = open_dataset(url)
+    nbytes = input_dataset.nbytes / 2
+    data_vars = input_dataset.data_vars
+    data_vars_with_time = (
+        []
+    )  # Only use data variables with a time component for pivoting in bisect request.
+    time_indices = (
```

Comment:
Just curious what happens if someone requests a file without a time index. (We have a few of these, geography files for hydrology, though at present users cannot request them directly and we have no plans to do that, they're only used for some statistical calculations in PCEX. Also they're pretty small and would not need to be bisected, so this is a big hypothetical).

Reply:
It'll just return the file on its own. If there were a hypothetical file without a time index, but it was bigger than 500 MB, then this wouldn't work because it doesn't have another variable to act as the "pivot".